docs/testnet/example-dapps.md

```diff
@@ -2,44 +2,37 @@
 1. MetaMask will open and ask to connect your account. Click `Next` then click `Connect`.
-1. Approve the payment of 1 token unit to play the game (this will be added to the prize pool) by clicking the `Approve game fee` button.
-1. MetaMask will ask for your account to sign a transaction specifying the Guess contract address as the delegate. Click `Confirm`. Once approved you will see a confirmation popup. Click `OK`.
+1. Approve the payment of 1 or more token units to play the game (this will be added to the prize pool) by clicking the `Approve game fee` button.
+1. MetaMask will ask for your account to sign a transaction specifying the Guess contract address as the approval delegate. This means that you're giving permission for the game to take the participation fee. Click `Confirm`. Once approved you will see a confirmation popup. Click `OK`.
 1. Now you can play the game by typing your guess into the secret number field. Click the `Submit guess` button.
 1. MetaMask will ask for your account to sign a contract interaction (your guess). Click `Confirm`. Note how the data presented by MetaMask to you is not yet encypted. That happens when MetaMask signs the transaction and sends it to the wallet extension "network", allowing the wallet extension to encrypt it before it leaves your computer.
 1. Open MetaMask to check the progress of your guess transaction in the `Activity` tab of MetaMask. Wait a few moments for it to change from pending to complete.
-1. Refresh the guessing game browser window / tab to see if the prize pool has either:
-    * Reset to 0. Congratulations! You won the game! Or,
-    * Incremented by 1. Unlucky! You did not guess the secret number. Please try again.
+1. The game should show you two messages, one confirming the success of the Approval transaction, and the second confirming the result of the guess attempt (whether correct, wrong, warmer or colder than the last guess).
 
 Once the guess transaction is completed you can check the guess transaction on ObscuroScan:
 1. In MetaMask click on the transaction to open it then click `Copy Transaction ID`. Open [ObscuroScan](http://testnet.obscuroscan.io/).
 1. Paste your copied transaction ID into the search box to find your individual guess transaction. Note how the transaction data visible to everyone is encrypted in the field `EncryptedTxBlob` and how (for now) the transaction is decrypted to allow developers to confirm correct behaviour of the network.
 1. You can see your guess as the number at the right hand end of the `input` value in a hexadecimal format, e.g. a guess of 99 is shown as 63.
 
 ### Known Issues and Limitations
-1. You requested game tokens using the Discord thread but there was no response.
+1. You have requested OBX tokens using the Discord thread but there was no response.
```

Comment:
Is this still relevant? Aren't you saying above that tokens are auto-allocated?

Reply:
Yep